### DISCUSSION TITLE
MVC(Model-View-Controller)와 Flux 모델의 차이점은 무엇인가요?

### DISCUSSION THREAD

**Discussion by lledellebell** [diving-react/study-react-deepdive#72] [General]
MVC와 Flux 모델의 가장 큰 차이점은 무엇인가요?

**Comment by skiende74** [comment]
MVC는
뷰가 모델을 바꿀수도있고, 모델이 뷰를 바꾸기도 하는

양방향바인딩이지만
Flux모델은
Action - Dispatcher - Model - View
일방향 바인딩입니다
